src/components/CharacterCard/CharacterCard.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import '/.CharacterCard.scss';
+
+const CharacterCard = () => {
+
+  return (
+    <section className="characterCard">
+      <h3 className="characterName"></h3>
+      <img className="favoriteStar"></img>
+      <p className="characterSpecies"></p>
+      <div></div>
+      <section className="characterPlanetInfo">
+        <img className="planetIcon"></img>
+        <p className="planetName"></p>
+        <p className="planetPopulation"></p>
+      </section>
+      <h4 className="characterMoviesHeader"></h4>
+      <ul>
+        {/* Movie list placeholder*/}
+      </ul>
+    </section>
```

Comment:
Can we add header tag for wrapping character name, div as wrapper for home description?

Reply:
Sounds good, will do!